packages/prismarine/src/Console.ts

```diff
@@ -74,19 +67,9 @@ export default class Console extends EntityLike implements Service {
 
             // Handle commands.
             if (input.startsWith('/')) {
-                void this.server.getCommandManager().dispatchCommand(this as any, this as any, input);
+                void this.server.getCommandManager().dispatchCommand(this, this as any, input);
                 return;
             }
-
-            void this.server.emit(
```

Comment:
This is somehow wrong, a console just executes commands: it should use the /say command in order to do  that.